TypeError: can't compare offset-naive and offset-aware datetimes

I am getting `TypeError: can't compare offset-naive and offset-aware datetimes` after updating to 0.12.0, for example:

```python
self = <quart_rate_limiter.RateLimiter object at 0x11652a010>, endpoint = 'graphql.graphql_server'
rate_limits = [RateLimit(count=50, period=datetime.timedelta(seconds=1), key_function=None, skip_function=None)]

    async def _raise_on_rejection(self, endpoint: str, rate_limits: List[RateLimit]) -> None:
        now = datetime.now(UTC)
        for rate_limit in rate_limits:
            key = await self._create_key(endpoint, rate_limit)
            # This is the GCRA rate limiting system and tat stands for
            # the theoretical arrival time.
>           tat = max(await self.store.get(key, now), now)
E           TypeError: can't compare offset-naive and offset-aware datetimes
```

I think this is related to the fact that `datetime.utcnow()` is naïve and `datetime.now(timezone.utc)` is timezone-aware (see 108f576a47cfa1a04c6f914407d7609ec40b017d):

```python
(Pdb) datetime.utcnow().tzinfo is None
True
(Pdb) datetime.now(timezone.utc).tzinfo
datetime.timezone.utc
```

Probably the value returned from storage is naïve: either because it was stored with version 0.11.0 or because the serializer/deserializer might ignore TZ info while writing/reading to the storage.

Maybe we need to check:
* Is stored datetime naive?
    * If so, assume UTC

If that sounds good, happy to open a PR : )

--- a/src/quart_rate_limiter/__init__.py
+++ b/src/quart_rate_limiter/__init__.py
@@ -288,7 +288,10 @@ class RateLimiter:
             key = await self._create_key(endpoint, rate_limit)
             # This is the GCRA rate limiting system and tat stands for
             # the theoretical arrival time.
-            tat = max(await self.store.get(key, now), now)
+            stored = await self.store.get(key, now)
+            if stored.tzinfo is None:
+                stored = stored.astimezone(UTC)
+            tat = max(stored, now)
             separation = (tat - now).total_seconds()
             max_interval = rate_limit.period.total_seconds() - rate_limit.inverse
             if separation > max_interval:
@@ -301,7 +304,10 @@ class RateLimiter:
         now = datetime.now(UTC)
         for rate_limit in rate_limits:
             key = await self._create_key(endpoint, rate_limit)
-            tat = max(await self.store.get(key, now), now)
+            stored = await self.store.get(key, now)
+            if stored.tzinfo is None:
+                stored = stored.astimezone(UTC)
+            tat = max(stored, now)
             new_tat = max(tat, now) + timedelta(seconds=rate_limit.inverse)
             await self.store.set(key, new_tat)
 
@@ -320,7 +326,10 @@ class RateLimiter:
         else:
             key = await self._create_key(endpoint, min_limit)
             now = datetime.now(UTC)
-            tat = max(await self.store.get(key, now), now)
+            stored = await self.store.get(key, now)
+            if stored.tzinfo is None:
+                stored = stored.astimezone(UTC)
+            tat = max(stored, now)
             separation = (tat - now).total_seconds()
             remaining = int((min_limit.period.total_seconds() - separation) / min_limit.inverse)
             response.headers["RateLimit-Limit"] = str(min_limit.count)

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -24,7 +24,7 @@ def _fixed_datetime(monkeypatch: MonkeyPatch) -> datetime:
     class MockDatetime(datetime):
         @classmethod
         def now(cls, tz) -> datetime:  # type: ignore
-            return datetime(2019, 3, 4)
+            return datetime(2019, 3, 4, tzinfo=tz)
 
     monkeypatch.setattr(quart_rate_limiter, "datetime", MockDatetime)
     return MockDatetime.now(timezone.utc)

